homeassistant/components/tailwind/entity.py

```diff
@@ -0,0 +1,25 @@
+"""Base entity for the Tailwind integration."""
+from __future__ import annotations
+
+from homeassistant.helpers.device_registry import CONNECTION_NETWORK_MAC, DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import TailwindDataUpdateCoordinator
+
+
+class TailwindEntity(CoordinatorEntity[TailwindDataUpdateCoordinator]):
+    """Defines an Tailwind entity."""
+
+    _attr_has_entity_name = True
+
+    def __init__(self, coordinator: TailwindDataUpdateCoordinator) -> None:
+        """Initialize an Tailwind entity."""
+        super().__init__(coordinator=coordinator)
```

Comment:
```suggestion
        super().__init__(coordinator)
```

Reply:
Sorry, I don't like positional arguments.

Reply:
Applied in [93aec43](https://github.com/home-assistant/core/pull/105926/commits/93aec43f1181b93c79d868dcb5571242259be00d)